colorcet/__init__.py

```diff
@@ -155,7 +170,7 @@ def name_sortfn(name):
     return ',  '.join(sorted(names, key=name_sortfn))
 
 
-def all_original_names(group=None, not_group=None, only_aliased=False, only_CET=False):
+def all_original_names(group=None, not_group=None, only_aliased=False, only_CET=False): # type: ignore[no-untyped-def] # temp fix
```

Comment:
Adding type hints to this function proved unexpectedly difficult so I skipped it for now...

Reply:
I would type those that are easy (if there are any) and use `Any` for the complicated type hints.